MODULE.bazel.lock

```diff
@@ -100,19 +100,19 @@
     "@@apple_support~//crosstool:setup.bzl%apple_cc_configure_extension": {
```

Comment:
Are the changes to this file mandatory? Why do you want to introduce these modifications?